packages/ragbits-document-search/src/ragbits/document_search/documents/document.py

```diff
@@ -32,13 +32,23 @@ class DocumentType(str, Enum):
     UNKNOWN = "unknown"
 
 
+class DocumentLocation(BaseModel):
+    """
+    An object representing position of chunk within document
+    """
+
+    page_number: int | None = None
+    coordinates: dict | None = None
+
+
 class DocumentMeta(BaseModel):
     """
     An object representing a document metadata.
     """
 
     document_type: DocumentType
     source: LocalFileSource | GCSSource | HuggingFaceSource = Field(..., discriminator="source_type")
+    location: DocumentLocation | None = None
```

Comment:
I think this should be defined in the `Element` class

Reply:
done